src/Microsoft.DotNet.ProjectJsonMigration/Report/ProjectMigrationReport.cs

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Collections.Generic;
+using System.Linq;
+namespace Microsoft.DotNet.ProjectJsonMigration
+{
+    public class ProjectMigrationReport
```

Comment:
Can we put a comment here that we need to update the xproj migration tools in the roslyn-project-system if this is ever updated?